project-base/composer.json

```diff
@@ -148,7 +148,7 @@
         "component-dir": "web/components",
         "sort-packages": true,
         "platform": {
-            "php": "7.2"
+            "php": "7.4.1"
```

Comment:
Do we still need platform to  be defined?

Reply:
It unifies the version of the packages used for developers (freshly built image can have a different version than the older php-fpm image, or natively installed php).
I agree it's now much less needed as we use only one minor version, but I still want to keep it. It's not doing any harm and it's consistent with the documentation.